sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Tests/E2ETests/AspNetCoreInstrumentationTests.cs

```diff
@@ -0,0 +1,215 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#if !NETFRAMEWORK
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Monitor.OpenTelemetry.Exporter.Models;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using OpenTelemetry.Instrumentation.AspNetCore;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+using Xunit;
+
+namespace Azure.Monitor.OpenTelemetry.AspNetCore.Tests.E2ETests
+{
+    [CollectionDefinition("Instrumentation.AspNetCore", DisableParallelization = true)]
+    [Collection("Instrumentation.AspNetCore")]
+    public class AspNetCoreInstrumentationTests
+        : IClassFixture<WebApplicationFactory<AspNetCoreTestApp.Program>>, IDisposable
+    {
+        private readonly WebApplicationFactory<AspNetCoreTestApp.Program> _factory;
+
+        private const string TestServiceName = nameof(TestServiceName), TestServiceNamespace = nameof(TestServiceNamespace), TestServiceInstance = nameof(TestServiceInstance), TestServiceVersion = nameof(TestServiceVersion);
```

Comment:
nit: consider placing in different lines for readability.

Reply:
i moved these to a static class since they're shared by multiple tests.